envs/environment.yaml

```diff
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
-name: pypsa-earth
+name: pypsa-earth-scip
```

Comment:
It appears like a local change pushed accidentally 😄
Could you please revert?

Reply:
😬

Reply:
Thank you.